app/bundles/EmailBundle/Helper/MailHelper.php

```diff
@@ -1264,7 +1264,13 @@ public function getIdHash()
     public function setIdHash($idHash = null, $statToBeGenerated = true)
     {
         if ($idHash === null) {
-            $idHash = uniqid();
+            /*
+            18.05.2018 - hammad-tfg: 
+                         fixed issue # 6093 by using more_entropy parameter to PHP uniqid. 
+                         Read more here: http://php.net/manual/en/function.uniqid.php
+                         Issue link: https://github.com/mautic/mautic/issues/6093
+            */
+            $idHash = str_replace('.','',uniqid('',true));//uniqid();
```

Comment:
Please remove the comment at the end of this row. The previous version of this row will be in the git history.